app.json

```diff
@@ -0,0 +1,23 @@
+{
+  "scripts": {
+    "postdeploy": "bundle exec rails db:seed",
+    "pr-predestroy": "bundle exec rake heroku:pr_destroy"
```

Comment:
Where does this rake task come from?

Reply:
Thanks for spotting this, it made it's way back in by mistake! New commit to remove this